<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dbgwidget</class>
 <widget class="QMainWindow" name="dbgwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Отладка модулей СУЛ</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_17">
      <property name="title">
       <string>Команды управления</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <property name="spacing">
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Абонент</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="abonentIndex">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>255</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Тип сообщения</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="abonentCmd"/>
         </item>
         <item>
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>Номер блока данных</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="msgBlockNum">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>16</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>Количество байт данных</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="msgBlockSize">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>128</number>
           </property>
           <property name="value">
            <number>128</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>Период сообщений, мс</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="dbgPeriod">
           <property name="maximum">
            <number>5000</number>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSend">
           <property name="text">
            <string>Старт</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnShowView">
           <property name="text">
            <string>Изображение</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_12">
      <property name="title">
       <string>Командный блок</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="headerLayout">
         <item>
          <widget class="QTableWidget" name="headerData">
           <row>
            <property name="text">
             <string>0</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>1</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>2</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>3</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>4</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>5</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>6</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>7</string>
            </property>
           </row>
           <column>
            <property name="text">
             <string>1</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QGroupBox" name="groupBox_13">
      <property name="title">
       <string>Передача данных</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="dataOutLayout">
         <item>
          <widget class="QTableWidget" name="outputData">
           <row>
            <property name="text">
             <string>0</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>1</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>2</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>3</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>4</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>5</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>6</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>7</string>
            </property>
           </row>
           <column>
            <property name="text">
             <string>1</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QGroupBox" name="groupBox_14">
      <property name="title">
       <string>Прием данных</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="dataInLayout">
         <item>
          <widget class="QTableWidget" name="inputData">
           <row>
            <property name="text">
             <string>0</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>1</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>2</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>3</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>4</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>5</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>6</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>7</string>
            </property>
           </row>
           <column>
            <property name="text">
             <string>1</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>620</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
